kafka/metrics/kafka_metric.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+
+class KafkaMetric(object):
+    def __init__(self, lock, metric_name, measurable, config):
+        if not metric_name:
+            raise ValueError('metric_name must be non-empty')
+        if not measurable:
+            raise ValueError('measurable must be non-empty')
+        self._metric_name = metric_name
+        self._lock = lock
+        self._measurable = measurable
+        self._config = config
+
+    @property
+    def metric_name(self):
+        return self._metric_name
+
+    @property
+    def measurable(self):
+        return self._measurable
+
+    def _get_config(self):
+        return self._config
+
+    def _set_config(self, config):
+        with self._lock:
```

Comment:
simple variable assignment is generally considered atomic in python. can we drop the lock here?